org/postgresql/jdbc4/AbstractJdbc4Connection.java

```diff
@@ -99,7 +99,7 @@ public Struct createStruct(String typeName, Object[] attributes) throws SQLExcep
     public Array createArrayOf(String typeName, Object[] elements) throws SQLException
     {
         checkClosed();
-        int oid = getTypeInfo().getPGArrayType(typeName);
+        int oid = getTypeInfo().getPGArrayType(typeName.toLowerCase());
```

Comment:
It should not be..... my mistake. How do you create two different PR's without committing